HISTORY.md

```diff
@@ -926,7 +926,7 @@
 * chore: git tag with release@semver (#8035)
 * fix(linux): Properly set context after changing IP (#8026)
 * chore(linux): use faster zero-length string check (#8037)
-* chore(linux): log failures to `km_kbp_context_clear(context)` (#8036)
+* chore(linux): log failures to `km_core_context_clear(context)` (#8036)
```

Comment:
Do we want to leave HISTORY.md intact? 
This PR would then add an entry for the rename.

Reply:
Yes my bad I backed out one change to HISTORY.md but missed the other one!

Reply:
I like this suggestion.